src/main/scala/com/microsoft/azure/iot/kafka/connect/source/IotHubSourceTask.scala

```diff
@@ -55,7 +58,7 @@ class IotHubSourceTask extends SourceTask with LazyLogging with JsonSerializatio
     val topic = props.get(IotHubSourceConfig.KafkaTopic)
     val batchSize = props.get(IotHubSourceConfig.BatchSize).toInt
     val startTime = getStartTime(props.get(IotHubSourceConfig.IotHubStartTime))
-    val eventHubName = props.get(IotHubSourceConfig.EventHubCompatibleName)
+    eventHubName = props.get(IotHubSourceConfig.EventHubCompatibleName)
```

Comment:
Better would be to add eventHubName to IotHubPartitionSource. That way it can be added to the logs in that class as well.